bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-glossary",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
as you're merging this into the Glossary-convert-to-es6-classes branch, there's no need to bump the version as that hasn't even been merged yet alone released